core/src/main/java/org/elasticsearch/common/unit/ByteSizeValue.java

```diff
@@ -107,23 +107,26 @@ public double getPbFrac() {
 
     @Override
     public String toString() {
-        long bytes = getBytes();
```

Comment:
Ok.
I though avoiding extra allocations just to call `toString()` would be better.
Static `parseBytesSizeValue` invites for the counterpart `formatBytesSizeValue`.